app/src/main/java/com/pajato/android/gamechat/exp/BaseExperienceFragment.java

```diff
@@ -182,6 +182,8 @@ public Experience getExperience() {
         super.onStart();
         if (mAdView != null && mLayout != null && type != null && type.expType == null)
             initAdView(mLayout);
+        // Initialize tile click handler
```

Comment:
This method essentially does two things.  A single comment can express that nicely while making the code itself grokable in a single glance.  For example,

    // Initialize ads when necessary and set up the tile click handler for the base game.